packages/@sanity/image-url/src/builder.js

```diff
@@ -47,10 +47,6 @@ class ImageUrlBuilder {
     return this._withOptions({focalPoint: {x, y}})
   }
 
-  fit(fit) {
-    return this._withOptions({fit})
-  }
-
```

Comment:
Removing the fit method surely wasn't intentional?

Reply:
There was a duplicate of this function